frontend-react/src/components/USLink.tsx

```diff
@@ -1,38 +1,116 @@
-import React, { PropsWithChildren } from "react";
+import React, { AnchorHTMLAttributes } from "react";
 import { Link, NavLink } from "react-router-dom";
 
 // Known issue with the `PropsWithChildren generic in React 18,
```

Comment:
Is this comment needed anymore?

Reply:
Ah, not _this_ wording, but I did mean to change it to reflect why we aren't using the `PropsWithChildren` type in general. Also not sure why I added `React.ReactNode[]` -- can't think of a time we'd need more than one element inside the link tag.